src/charm.py

```diff
@@ -225,6 +246,13 @@ def _on_config_changed(self, event: EventBase):
             event.defer()
             return
         self._on_bessd_pebble_ready(event)
+        if self._restart_pod:
+            try:
+                self._kubernetes_multus.delete_pod()
```

Comment:
Would it be ok to just do
`self._kubernetes_multus.client.delete(Pod, pod_name, namespace)`
here instead of modifying the lib?

Reply:
I think we require to reuse the delete method also in other charms which uses multus. Adding it to the library seemed reasonable to me as it is easy to manage.